src/main/java/com/cottonfactory/products/controllers/ShoeController.java

```diff
@@ -0,0 +1,24 @@
+package com.cottonfactory.products.controllers;
+
+import com.cottonfactory.products.entities.ShoeProduct;
+import com.cottonfactory.products.services.ShoeService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ShoeController {
+
+    @Autowired
+    private ShoeService shoeService;
+
+    @PostMapping("/api/products/shoe")
```

Comment:
```suggestion
    @PostMapping("/api/v1/products/shoe")
```
the Product API home has an initial value as api/v1/products, probably a good idea to follow same versioning

Reply:
v1 is not required so not making any change